corehq/apps/sso/utils.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+
+
+def get_request_data(request):
+    """
+    OneLogin's python3-saml library expects a very specifically formatted
+    "request_data" object as it is framework agnostic and each framework
+    (eg flask, django, tornado) has its own way of populating the built in
+    request object
+    :param request:
+    :return: dictionary with fields that python3-saml expects
+    """
+    return {
+        'https': 'on' if request.is_secure() else 'off',
```

Comment:
I thought we always require SSL. Why on earth would an authentication provider use plain HTTP without SSL these days?

I can understand why we may want to allow having it turned off in development environments (which I assume would be indicated by `settings.SAML2_DEBUG`). Can we use that setting rather than `request.is_secure()`?

Reply:
sure! will make that update

Reply:
done ✅  @millerdev